media-video/ffmpeg/files/vmaf-models-default-path.patch

```diff
@@ -0,0 +1,13 @@
+# Put default path of models to /usr/share/vmaf/model
```

Comment:
I don't think the leading `#` is necessary here, but I don't know if it's convention to add comments to patches like this.

Reply:
It's true that it's rather obvious what the patch does, do you suggest to remove the comment altogether ?

Reply:
No, just the `#` prefix. I love comments :-)

Reply:
Got it, done!